modules/op-resource-quota-for-pipelines.adoc

```diff
@@ -0,0 +1,205 @@
+// This module is included in the following assembly:
+//
+// */cicd/pipelines/reducing-pipelines-resource-consumption.adoc
+
+[id="resource-quota-for-pipelines_{context}"]
+= Resource quota for pipelines
+
+A resource quota limits the total resource consumption per namespace. It can limit the quantity of objects created in a namespace, based on the type of the object. In addition, it can limit the total amount of compute resources consumed by resources in that namespace, using a compute resource quota.
+
+Currently, Kubernetes does not provide any direct means of specifying compute resource quota for pipelines. You can implement a workaround for this limitation by using resource quota per priority class.
+
+[id="specifying-pipelines-resource-quota-using-priority-class_{context}"]
+== Specifying pipelines resource quota using priority class
+
+A priority class defines a mapping from a priority class name to the integer value of the priority, with higher value implying the higher the priority. After you create a priority class, you can create pods that specify the priority class name in their specifications. In addition, you can control a pod's consumption of system resources based on the pod's priority.
+
+Specifying resource quota for a pipeline is approximately equivalent to setting resource quota for the subset of pods created by a pipeline run. The following steps provide an example of the workaround by specifying resource quota based on priority class.
+
```

Comment:
Need to add this note here - https://github.com/openshift/openshift-docs/pull/38489#discussion_r751986657